04-Parallel-Simulations-With-MPI/03-Using-Snapshots-With-MPI.ipynb

```diff
@@ -0,0 +1,296 @@
+{
```

Comment:
<p>I think there is a typo there and there is an extra word "make" before the word specify.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/43/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Fixed.